crates/primitives/src/transaction/signature.rs

```diff
@@ -157,7 +157,7 @@ mod tests {
 
     #[test]
     fn test_payload_len_with_eip155_chain_id() {
-        let signature = Signature { r: U256::default(), s: U256::default(), odd_y_parity: false };
+        let signature = Signature { r: U256::from(1), s: U256::from(1), odd_y_parity: false };
```

Comment:
could you explain briefly why the change from 0 to 1 here?

Reply:
This is specifically to account for the new `Signature.v()` logic which returns `0` if `r` and `s` are both `0`
https://github.com/anton-rs/op-reth/blob/3cc5b222204a4a1143c8aea6651f3f361b665e5f/crates/primitives/src/transaction/signature.rs#L65-L67

Reply:
Happy to add this as an inline comment if this is deemed appropriate

Reply:
please, do. we'll be incredibly meticulous about the changes outside of the feature flag when reviewing the final PR

Reply:
Updated 👍